### DISCUSSION TITLE
Welcome to memray Discussions!

### DISCUSSION THREAD

**Discussion by pablogsal** [bloomberg/memray#54] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by cemilakkoc** [comment]
Are there unicorn hunters out there? Like for real.

**Comment by rochakgupta** [comment]
Thanks for developing and open sourcing this awesome tool!

**Comment by ghost** [comment]
How to use memary with Gunicorn and Django?
I have a project with Django and I did a multiread with Gunicorn. but my project has a memory leak. I want to monitor memory with "memary" but I don't know how to use "memary".

**Reply by godlygeek** [reply]
This is being discussed in #56

**Comment by piskov** [comment]
Could anyone please identify the font used in the “[Live mode](https://github.com/bloomberg/memray#live-mode) ” screenshots?

![D0D0C6A6-D448-46F5-A30E-C9B98F3A7674](https://user-images.githubusercontent.com/4349021/169528496-215acdb3-1f98-4cf5-bee7-977f4e8ff750.jpeg)

**Reply by pablogsal** [reply]
> Could anyone please identify the font used in the “[Live mode](https://github.com/bloomberg/memray#live-mode) ” screenshots?

This is mononoki Nerd font, available here:

https://github.com/ryanoasis/nerd-fonts/tree/master/src/unpatched-fonts/Mononoki

**Comment by ziyoujiyi** [comment]
I hope to develop from memray, can you give me more docs to descripe its mechanism？
